test/Spec.hs

```diff
@@ -1,10 +1,58 @@
+import Data.List
+import Data.Function
 import Boids
 import Test.HUnit
-import Test.HUnit.Approx
 import System.Exit
 
+testIsWithinView :: Test
+testIsWithinView  = TestCase (assertBool "Seeing a boid behind me" (not $ isWithinView boid1 boid2))
+    where boid1 = Boid { boidPosition = (0.0, 0.0)
+                       , boidHeading = 0.0
+                       , boidSteer = 0.0
+                       }
+          boid2 = Boid { boidPosition = (-10.0, 0.0)
+                       , boidHeading = 0.0
+                       , boidSteer = 0.0
+                       }
+
+testGetNearBoids :: Test
+testGetNearBoids = TestCase (assertEqual "Got unexpected nearest boids" (sortBoids nearestBoids) (sortBoids actualResult))
+    where pivotBoid = Boid { boidPosition = (388.46451, 289.70023)
+                           , boidHeading = -pi / 2.0
+                           , boidSteer = 0.0
+                           }
+          nearestBoids = [ Boid { boidPosition = (291.93515, 228.0032)
+                                , boidHeading = 0.0
+                                , boidSteer = 0.0
+                                }
+                         , Boid { boidPosition = (392.85715, 127.36221)
+                                , boidHeading = 0.0
+                                , boidSteer = 0.0
+                                }
+                         ]
+          farestBoids = [ Boid { boidPosition = (505.70178, 58.542339)
```

Comment:
Superlative adjectives for "far" [are](https://en.wiktionary.org/wiki/far#Adjective): "farthest", "furthest", "farthermost", "furthermost".